src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -551,7 +552,11 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		this._onDidGroupChange.fire({ kind: GroupChangeKind.EDITOR_STICKY, editor });
 	}
 
-	private onDidOpenEditor(editor: EditorInput): void {
+	private onDidMoveEditor(event: EditorMoveEvent): void {
+		this._onDidGroupChange.fire({ kind: GroupChangeKind.EDITOR_MOVE, editor: event.editor, previousEditorIndex: event.index, editorIndex: event.target });
```

Comment:
@bpasero  Added here

Reply:
That seems wrong? `editorIndex: event.target` is not correct or is it? `event.target` is the group identifier, not the index of the tab.